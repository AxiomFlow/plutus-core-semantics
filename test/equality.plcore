(module Foo (imported) (exported () ()) (local () ())
  (define r1 (builtin equalsInt (2 2)))
  (define r2 (builtin equalsInt (2 3)))
  (define r3 (builtin equalsFloat (2.0 2.0)))
  (define r4 (builtin equalsFloat (2.0 3.0)))
  (define r5 (builtin equalsByteString (#AAA #AAA)))
  (define r6 (builtin equalsByteString (#AAA #111)))
)
