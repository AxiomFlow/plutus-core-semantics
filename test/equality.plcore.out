<T> <k> . </k> <env> ( Foo . r4 ) |-> ( con Nat . False .ValList ) ( Foo . r5 ) |-> ( con Nat . True .ValList ) ( Foo . r2 ) |-> ( con Nat . False .ValList ) ( Foo . r3 ) |-> ( con Nat . True .ValList ) ( Foo . r1 ) |-> ( con Nat . True .ValList ) ( Foo . r6 ) |-> ( con Nat . False .ValList ) </env> <mod> Foo </mod> </T>
